src/lib/pages/backoffice/ILL/BorrowingRequest/BorrowingRequestEditor/BorrowingRequestEditor.js

```diff
@@ -17,6 +18,20 @@ export class BorrowingRequestEditor extends Component {
     }
   }
 
+  get borrowingRequest() {
```

Comment:
very minor:
```suggestion
  get initialData() {
```
WDYT?